File: CUSTOR.EIC.API/src/app/model/Country.ts
export class CountryModel {
  public Amharic: string;
  public English: string;
  public Id: number;

  constructor(Amharic?: string, English?: string,Id?: number) {
    this.Id = Id;
    this.Amharic = Amharic;
    this.English = English;
  }
}

File: CUSTOR.EICOnline.DAL/EntityLayer/Country.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace CUSTOR.EICOnline.DAL.EntityLayer
{
    public class Country
    {
        [Key] 
        public int Id { get; set; }
        public string English { get; set; }
        public string Amharic { get; set; }
    }
    public class CountryDTO
    {
        [Key] 
        public int Id { get; set; }
        public string English { get; set; }
        public string Amharic { get; set; }
        public int  InvestorId { get; set; }
        
        
    }
    
}